### ISSUE TITLE
conda creates an empty environment despite a spec

### ISSUE THREAD

**Issue by mcg1969** [conda/conda#8799]
## Current Behavior

conda creates an empty environment even though a specific package is supplied in the `create` command.

### Steps to Reproduce

```
conda create -n testnp numpy=1.11.2
```

## Expected Behavior

Either numpy and its dependencies are installed, or an error is raised saying that it can't do so.

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```

     active environment : base
    active env location : /Users/mgrant/miniconda3
            shell level : 1
       user config file : /Users/mgrant/.condarc
 populated config files : /Users/mgrant/.condarc
          conda version : 4.6.14
    conda-build version : 3.17.8
         python version : 3.6.8.final.0
       base environment : /Users/mgrant/miniconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://conda.anaconda.org/mcg/osx-64
                          https://conda.anaconda.org/mcg/noarch
                          https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /Users/mgrant/miniconda3/pkgs
                          /Users/mgrant/.conda/pkgs
       envs directories : /Users/mgrant/miniconda3/envs
                          /Users/mgrant/.conda/envs
               platform : osx-64
             user-agent : conda/4.6.14 requests/2.19.1 CPython/3.6.8 Darwin/18.6.0 OSX/10.14.5
                UID:GID : 502:20
             netrc file : None
           offline mode : False

```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
==> /Users/mgrant/.condarc <==
channel_priority: strict
channels:
  - defaults
  - mcg
  - conda-forge
```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
# packages in environment at /Users/mgrant/miniconda3:
#
# Name                    Version                   Build  Channel
ae-preflight              0.1.4                     dev_0    <develop>
anaconda-client           1.7.2                    py36_0    defaults
anaconda-enterprise-cli   5.3.0.4                   dev_0    <develop>
anaconda-project          0.8.2                    py36_0    defaults
appnope                   0.1.0            py36hf537a9a_0    defaults
arrow                     0.13.1                   py36_0    defaults
asn1crypto                0.24.0                   py36_0    defaults
backcall                  0.1.0                    py36_0    defaults
beautifulsoup4            4.7.1                    py36_1    defaults
binaryornot               0.4.4                      py_1    conda-forge
blas                      1.0                    openblas    defaults
bleach                    3.1.0                    py36_0    defaults
bzip2                     1.0.6                h1de35cc_5    defaults
ca-certificates           2019.5.15                     0    defaults
certifi                   2019.3.9                 py36_0    defaults
cffi                      1.12.1           py36hb5b8e2f_0    defaults
chardet                   3.0.4                    py36_1    defaults
click                     6.7              py36hec950be_0    defaults
click-repl                0.1.6                      py_0    mcg
clyent                    1.2.2                    py36_1    defaults
conda                     4.6.14                   py36_0    defaults
conda-build               3.17.8                   py36_0    defaults
conda-env                 2.6.0                         1    defaults
conda-verify              3.1.1                    py36_0    defaults
cookiecutter              1.6.0                 py36_1000    conda-forge
cryptography              2.5              py36ha12b0ac_0    defaults
dbus                      1.13.6               h90a0687_0    defaults
decorator                 4.3.2                    py36_0    defaults
defusedxml                0.6.0                      py_0    defaults
distro                    1.4.0                      py_0    conda-forge
entrypoints               0.3                      py36_0    defaults
expat                     2.2.6                h0a44026_0    defaults
filelock                  3.0.10                   py36_0    defaults
future                    0.17.1                   py36_0    defaults
gettext                   0.19.8.1             h15daf44_3    defaults
glib                      2.56.2               hd9629dc_0    defaults
glob2                     0.6                      py36_1    defaults
icu                       58.2                 h4b95b61_1    defaults
idna                      2.7                      py36_0    defaults
ipykernel                 5.1.0            py36h39e3cac_0    defaults
ipython                   7.3.0            py36h39e3cac_0    defaults
ipython_genutils          0.2.0            py36h241746c_0    defaults
ipywidgets                7.4.2                    py36_0    defaults
jedi                      0.13.3                   py36_0    defaults
jinja2                    2.10                     py36_0    defaults
jinja2-time               0.2.0                      py_2    conda-forge
jpeg                      9b                   he5867d9_2    defaults
jsonschema                2.6.0            py36hb385e00_0    defaults
jupyter                   1.0.0                    py36_7    defaults
jupyter_client            5.2.4                    py36_0    defaults
jupyter_console           6.0.0                    py36_0    defaults
jupyter_core              4.4.0                    py36_0    defaults
libarchive                3.3.3                h786848e_5    defaults
libcxx                    4.0.1                hcfea43d_1    defaults
libcxxabi                 4.0.1                hcfea43d_1    defaults
libedit                   3.1.20181209         hb402a30_0    defaults
libffi                    3.2.1                h475c297_4    defaults
libgfortran               3.0.1                h93005f0_2    defaults
libiconv                  1.15                 hdd342a3_7    defaults
liblief                   0.9.0                h2a1bed3_2    defaults
libopenblas               0.3.6                hdc02c5d_0    defaults
libpng                    1.6.37               ha441bb4_0    defaults
libsodium                 1.0.16               h3efe00b_0    defaults
libxml2                   2.9.9                hab757c2_0    defaults
lz4-c                     1.8.1.2              h1de35cc_0    defaults
lzo                       2.10                 h362108e_2    defaults
make                      4.2.1                h3efe00b_1    defaults
markupsafe                1.1.1            py36h1de35cc_0    defaults
mistune                   0.8.4            py36h1de35cc_0    defaults
nbconvert                 5.5.0                      py_0    defaults
nbformat                  4.4.0            py36h827af21_0    defaults
ncurses                   6.1                  h0a44026_1    defaults
nomkl                     3.0                           0    defaults
nose                      1.3.7                    py36_2    defaults
notebook                  5.7.8                    py36_0    defaults
numpy                     1.16.4           py36h926163e_0    defaults
numpy-base                1.16.4           py36ha711998_0    defaults
openssl                   1.1.1c               h1de35cc_1    defaults
pandas                    0.24.2           py36h0a44026_0    defaults
pandoc                    2.2.3.2                       0    defaults
pandocfilters             1.4.2                    py36_1    defaults
parso                     0.3.4                    py36_0    defaults
pcre                      8.43                 h0a44026_0    defaults
pexpect                   4.6.0                    py36_0    defaults
pickleshare               0.7.5                    py36_0    defaults
pip                       19.0.3                   py36_0    defaults
pkginfo                   1.5.0.1                  py36_0    defaults
poyo                      0.4.2                      py_0    conda-forge
prometheus_client         0.6.0                    py36_0    defaults
prompt_toolkit            2.0.9                    py36_0    defaults
psutil                    5.5.0            py36h1de35cc_0    defaults
ptyprocess                0.6.0                    py36_0    defaults
py-lief                   0.9.0            py36h1413db1_2    defaults
pycosat                   0.6.3            py36h1de35cc_0    defaults
pycparser                 2.19                     py36_0    defaults
pygments                  2.3.1                    py36_0    defaults
pyopenssl                 19.0.0                   py36_0    defaults
pyqt                      5.9.2            py36h655552a_2    defaults
pysocks                   1.6.8                    py36_0    defaults
python                    3.6.8                haf84260_0    defaults
python-dateutil           2.8.0                    py36_0    defaults
python-libarchive-c       2.8                      py36_6    defaults
python.app                2                        py36_9    defaults
pytz                      2018.9                   py36_0    defaults
pyyaml                    3.13             py36h1de35cc_0    defaults
pyzmq                     18.0.0           py36h0a44026_0    defaults
qt                        5.9.7                h468cd18_1    defaults
qtconsole                 4.5.1                      py_0    defaults
readline                  7.0                  h1de35cc_5    defaults
requests                  2.19.1                   py36_0    defaults
ruamel_yaml               0.15.46          py36h1de35cc_0    defaults
send2trash                1.5.0                    py36_0    defaults
setuptools                40.8.0                   py36_0    defaults
sip                       4.19.8           py36h0a44026_0    defaults
six                       1.12.0                   py36_0    defaults
soupsieve                 1.7.1                    py36_0    defaults
sqlite                    3.26.0               ha441bb4_0    defaults
terminado                 0.8.2                    py36_0    defaults
testpath                  0.4.2                    py36_0    defaults
tk                        8.6.8                ha441bb4_0    defaults
tornado                   6.0.1            py36h1de35cc_0    defaults
tqdm                      4.31.1                     py_0    defaults
traitlets                 4.3.2            py36h65bd3ce_0    defaults
urllib3                   1.23                     py36_0    defaults
versioneer                0.18                       py_1    conda-forge
wcwidth                   0.1.7            py36h8c6ec74_0    defaults
webencodings              0.5.1                    py36_1    defaults
wheel                     0.33.1                   py36_0    defaults
whichcraft                0.5.2                      py_1    conda-forge
widgetsnbextension        3.4.2                    py36_0    defaults
xz                        5.2.4                h1de35cc_4    defaults
yaml                      0.1.7                hc338f04_2    defaults
zeromq                    4.3.1                h0a44026_3    defaults
zlib                      1.2.11               h1de35cc_3    defaults
zstd                      1.3.7                h5bba6e5_0    defaults
```
</p></details>

**Comment by jjhelmus** [comment]
I'm not able to reproduce this on my machine.  Does adding `-vvv` show any information about what conda is doing?

**Comment by jjhelmus** [comment]
Wait, I am seeing strange behavior when `channel_priority: strict` is added, removing it results in the expected results.

**Comment by msarahan** [comment]
I think this is fixed in conda 4.7.4.  With and without strict, I see:

```
conda create -n testnp numpy=1.11.2
Collecting package metadata (current_repodata.json): done
Solving environment: failed
Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - numpy=1.11.2

Current channels:

  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

Closing, but please reopen if it's not actually fixed.
